devtool/client/src/components/DevToolsPanel.tsx

```
@@ -0,0 +1,115 @@
+    return (
+        <div className={`devtools-container position-${finalConfig.position}`}>
+            {isOpen && (
+                <div className="devtools-panel">
+                    <div className="devtools-header">
+                        <h3>🔧 DevTools</h3>
+                        <button
+                            className="devtools-close"
+                            onClick={() => setIsOpen(false)}
+                            title="Close"
+                        >
+                            ×
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)